### PR TITLE
feat: OL Surge PreFare Slots

### PR DESCRIPTION
**Pull request by cmaddox5**
**Asana task**: [Add new slots to pre-fare template](https://app.asana.com/0/1185117109217413/1202809139605053/f)

Added a new slot for OL Surge widgets: `orange_line_surge_upper` and `orange_line_surge_lower`. Right now, each slot takes up half of the space under the header. If any widget in the `body_right_normal` group has a higher priority, it will not show these new widgets.

- [ ] Tests added?

### CODE REVIEW

File: assets/css/v2/pre_fare/body_right/surge.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  left: 0;
+
+  width: 1080px;
+  height: 876px;
+  overflow: hidden;
+}
+
+.body-right-surge__lower {
+  position: absolute;
+  top: 918px;
```
Comment (jzimbel-mbta) [inline assets/css/v2/pre_fare/body_right/surge.scss:19]: Should this be `876px`? I think normally we put one slight immediately next to (or below) the previous one, so I would have expected this `top:` to have the same value as the `height:` of the upper slot.

File: assets/css/v2/pre_fare/body_right/surge.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  left: 0;
+
+  width: 1080px;
+  height: 876px;
```
Comment (jzimbel-mbta) [inline assets/css/v2/pre_fare/body_right/surge.scss:23]: I think this height and that of the upper slot should add up to the body total height. Currently we have 876 + 876 = 1752 ≠ 1648 (assuming the comment at the top of the file is accurate)

**Review by jzimbel-mbta — COMMENTED** [review]
Just reviewing this because I need it for the BlueBikes widget 😅

File: assets/css/v2/pre_fare/body_right/surge.scss (hunk 1 above)
Comment (cmaddox5) [reply assets/css/v2/pre_fare/body_right/surge.scss:19]: Thanks for bumping this. I started to add some numbers initially until I realized we didn't know dimensions yet. Now that we have them, I'll make these match the final numbers in the designs.

File: assets/css/v2/pre_fare/body_right/surge.scss (hunk 2 above)
Comment (jzimbel-mbta) [reply assets/css/v2/pre_fare/body_right/surge.scss:23]: Ah that comment was not correct! The pre-fare screen body is 1720px tall.

Comment (cmaddox5) [reply assets/css/v2/pre_fare/body_right/surge.scss]: Yep. I originally copied this file from `normal.scss`, which also happened to be incorrect 😅 Got them both corrected.

**Review by jzimbel-mbta — APPROVED** [review]

**Pull request merged**